docs/2.9.0/docs/daml-finance/instruments/structured-product.rst

```diff
@@ -64,6 +64,53 @@ For this to work, you need to define an *Observation* as well:
 Since this option instrument is cash-settled, the underlying asset will not change hands. Instead,
 the cash value of the payoff is paid to the BRC holder.
 
+Auto-Callable
+=============
+
+The
+:ref:`AutoCallable <module-daml-finance-interface-instrument-structuredproduct-autocallable-instrument-66988>`
+instrument models auto-callable notes that pay a conditional coupon. At maturity, the principal
+amount is repaid unless a final barrier has been breached (in which case the performance of the
+underlying is paid). In other words, it is an AutoCallable Barrier Reverse Convertible where the KI
+barrier is observed at maturity.
+
+This is a single-underlying instrument. If the underlying closes above the call barrier on an
+observation date, the instrument is automatically redeemed early at the end of that period.
+
```

Comment:
I think we should clarify when the coupon is paid and when it is not paid. 

I would suggest to use sub-sections similar to how a term-sheet for this products is structured. Such as, 

- Redemption at maturity
- Coupon payment
- Early redemption
- Example